exercises/concept/welcome-to-tech-palace/.docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+# Introduction
+
+A `string` in Go is an immutable sequence of bytes, which don't necessarily have to represent characters.
+
+A `string` literal is a sequence of bytes enclosed in double quotes:
+
+```go
+firstName := "Jane"
+```
+
+Strings can be concatenated via the `+` operator:
+
+```go
+fullName := "Jane" + " " + "Austen"
+```
+
+Some special characters need to be escaped with a leading backspace:
+
+```go
+question := "\t or spaces?"
+```
+
+The `strings` package contains many useful functions to work on strings.
+
+```go
+import "strings"
+
+strings.ToLower("TEST") // Output: "test"
+```
```

Comment:
@ErikSchierboom Without a list of more functions or a link to where to find more functions, the info in the introduction is currently not sufficient to solve the exercise.

Reply:
I disagree with this statement for the following reason. One of the core ideas of Concept Exercises was to have students figure things out by themselves as much as possible. The way I intended this to work is to provide _just_ enough context to allow the student to Google things by themselves, but to always provide the hints as a fallback should the student not manage to figure things out. Applying that to this example, we've explained that there is a `strings` package (letting the student learn its exact name, useful for Googling), shown how to import it and how to use one example function (purposefully _not_ one of the functions the student requires).

I know not everyone is comfortable with the above approach, but it _is_ the original idea for how to introduce concepts in Concept Exercises. If you as Go maintainers would prefer to use a different approach, I'm okay with that, but I think there's great value in the above approach.

Reply:
I am ok either way, I was just a little confused because I had the feeling you were pushing a lot for providing all the needed info in other exercises. I got the wrong impression from that. Just ignore the comment.

Reply:
No problem. I will get this wrong from time to time too, so 🤷